pkg/appmanager/as3Manager.go

```diff
@@ -0,0 +1,100 @@
+package appmanager
+
+import (
+	"encoding/json"
+
+	log "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger"
+)
+
+type as3Template string
+type as3Declaration string
+
+type pool []Member
+type serviceName string
+type appName string
+type tenantName string
+type tenant map[appName][]serviceName
+type as3Object map[tenantName]tenant
+
+// Fake Function to validate the buildAS3Delaration method
+func getEndpoints(tenantName, appName, serviceName) pool {
+	//ip := []poolMember{"172.16.1.1", "192.16.2.2", "1.1.1.1"}
+	//return ip
+	return []Member{
+		{"1.1.1.1", 80, ""},
+		{"2.2.2.2", 80, ""},
+		{"3.3.3.3", 80, ""},
+	}
+}
+
+// Traverses through the AS3 Json using the information passed from buildAS3Declaration
+// Parses the AS3 Json and populates it with poolmembers
+func updatePoolMembers(tnt tenantName, app appName, svc serviceName, ips []string, port int32, as3Json map[string]interface{}) map[string]interface{} {
```

Comment:
as3Json can be renamed to a name with lesser ambiguity.

Reply:
Will make this as a new commit instead of a inline